roles/wp-cli/defaults/main.yml

```diff
@@ -1,2 +1,4 @@
 phar_url: https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar
 bin_path: /usr/bin/wp
+wp_cli_completion_url: https://raw.githubusercontent.com/wp-cli/wp-cli/master/utils/wp-completion.bash
```

Comment:
I see. But bash completion is the same for both AFAIK (I think is the the same script since I started using WP-CLI some 2 years ago or so).